assets/src/edit-story/app/font/fontProvider.js

```diff
@@ -53,13 +68,46 @@ function FontProvider({ children }) {
 
   const maybeEnqueueFontStyle = useLoadFontFiles();
 
+  const getMenuFonts = useCallback((fontFamilyList) => {
+    const newFontList = fontFamilyList.filter(
+      (fontName) =>
+        loadedFontFamily.current.findIndex((name) => name === fontName) < 0
```

Comment:
Is this the same as `loadedFontFamily.current.indexOf(fontName) < 0`?

Reply:
Correct! Thank you